sycl/test/basic_tests/handler/run_on_host_intel.cpp

```diff
@@ -1,6 +1,7 @@
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
+// XFAIL: windows
```

Comment:
Just to clarify: is this temporary solution to avoid failures in CI?
This feature doesn't seem to be Linux specific. Right?

If so, could you add a TODO comment, please?

Reply:
Done.